CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## [1.2.2](https://github.com/ably/ably-flutter/tree/v1.2.2)
+
+This release adds support for push notifications, including device activation with Ably (including device registration with APNs / FCM), subscribing devices for push notifications and handling the push notifications in your Flutter application, without writing iOS or Android code. Check out the dedicated documentation (`PushNotifications.md`) and the example app (`push_notification_service.dart`) for more information. We also fixed some bugs that were either reported in Github issues or found internally.
+
+[Full Changelog](https://github.com/ably/ably-flutter/compare/v1.2.2-preview.1...v1.2.2)
```

Comment:
@QuintinWillison It might be better to change this to `v1.2.2...v1.2.2`, to include the changes from the preview release too. What do you think?

Reply:
I would rather not make that call right now, under pressure to release. Please leave it as is. We can discuss later strategy for change logs where pre-release is in this mix.